### DISCUSSION TITLE
Comments on money payments

### DISCUSSION THREAD

**Discussion by abhisantos** [Zrips/CMI#7614] [Enhancement]
Please add a possibility for players and admins to add a comment when paying money.
There are already some plugins on spigotmc that have this. But i think it would be nice if CMI had.

For example, /pay [playername] [amount] [comments]
For admin it would be /money give [playername] [amount] [comments]

Thank you

**Comment by Mrkillero4ka** [comment]
I am for!
Or you can do the **/mpay** command, whoever wants to write comments can use this command, and who doesn't want to can just write **/pay**
